gef.py

```diff
@@ -1287,7 +1287,10 @@ def gdb_disassemble(start_pc, **kwargs):
         asm = insn["asm"].rstrip().split(None, 1)
         if len(asm) > 1:
             mnemo, operands = asm
-            operands = operands.split(",")
+            operands_and_comment = operands.split("//")
```

Comment:
Mind splitting this into another PR? I think this part can be improved.

Reply:
Thank you for review. Ok. I reverted that.